docs/azure/integrations-overview.md

```diff
@@ -104,6 +75,183 @@ The same is true for SQL and PostgreSQL services:
+
+serviceBus.AddServiceBusQueue("queue");
+```
+
+The preceding code:
+
+- Creates a new `builder` instance.
+- Adds a parameter named `existingServiceBusName` to the builder.
+- Adds an Azure Service Bus resource named `messaging` to the builder.
+- Calls the `PublishAsExisting` method on the `serviceBus` resource builder, passing the `existingServiceBusName` parameter—alternatively, you can use the `string` parameter overload.
+- Adds a queue named `queue` to the `serviceBus` resource.
+
+By default, the Service Bus parameter reference is assumed to be in the same Azure resource group. However, if it's in a different resource group, you can pass the resource group explicitly as a parameter to correctly specify the appropriate resource grouping.
+
+After the app host is executed in publish mode, the generated manifest file will include the `existingResourceName` parameter, which can be used to reference the existing Azure resource. Consider the following generated manifest file:
+
+```json
+{
+  "type": "azure.bicep.v0",
+  "connectionString": "{messaging.outputs.serviceBusEndpoint}",
+  "path": "messaging.module.bicep",
+  "params": {
+    "existingResourceName": "{existingResourceName.value}",
+    "principalType": "",
+    "principalId": ""
+  }
+}
+```
+
+For more information on the manifest file, see [.NET Aspire manifest format for deployment tool builders](../deployment/manifest-format.md).
+
+Additionally, the generated Bicep template includes the `existingResourceName` parameter, which can be used to reference the existing Azure resource. Consider the following generated Bicep template:
+
+```bicep
+@description('The location for the resource(s) to be deployed.')
+param location string = resourceGroup().location
+
+param existingResourceName string
+param principalType string
+param principalId string
+
+resource messaging 'Microsoft.ServiceBus/namespaces@2024-01-01' existing = {
+  name: existingResourceName
+}
+
+output serviceBusEndpoint string = messaging.properties.serviceBusEndpoint
+```
+
+For more information on the generated Bicep template, see [Infrastructure as code](#infrastructure-as-code).
+
+> [!WARNING]
+> When interacting with existing resources that require authentication, ensure the authentication strategy that you're configuring in the .NET Aspire application model aligns with the authentication strategy allowed by the existing resource. For example, it's not possible to use managed identity against an existing Azure PostgreSQL resource that isn't configured to allow managed identity. Similarly, if an existing Azure Redis resource disabled access keys, it's not possible to use access key authentication.
+
+### Configure existing Azure resources
+
+The `AsExisting` method is used when the distributed application is running in "run" or "publish" mode. Because the `AsExisting` method can operate in both scenarios, it can only support a parameterized reference to the resource name or resource group name.
```

Comment:
```suggestion
The `AsExisting` method is used when the distributed application is running in "run" or "publish" mode. Because the `AsExisting` method operates in both scenarios, it only supports a parameterized reference to the resource name or resource group name. This approach helps prevent the use of the same resource in both testing and production environments.
```